### ISSUE TITLE
Temporary Armour, Creates Infinitely Stacking "Value" Bug

### ISSUE THREAD

**Issue by Yyffii** [Interkarma/daggerfall-unity#2640]
**Describe the issue**

When 'Temporary Items' (Create Item Spell Effect) expire while equipped the armour value is retained, this leads to an infinitely stackable
armour value

**To Reproduce**

simply create a 1/1 duration create item spell, and equip then let expire armour, Repeat.

**Expected behavior**

the value should reset to the value of no armour in that slot since it has been "unequipped"

**Screenshots and Logs**

![image](https://github.com/Interkarma/daggerfall-unity/assets/46986147/ba693469-793b-4b59-ae4f-204684fbd0bd)
![image](https://github.com/Interkarma/daggerfall-unity/assets/46986147/77c6a7e6-4659-4fe2-bd0e-de30bca14d92)
![image](https://github.com/Interkarma/daggerfall-unity/assets/46986147/9dc86df9-abea-4f6d-83d5-6a67786e1dff)

**Desktop (please complete the following information):**

 - OS: Windows 11 64bit
 - Version 1.1.0 RC2

**Additional context**

issue doesn't inherently lie within the create item script
but any temporary item.

issue is possibly from DaggerfallWorkshop.Game.Items.ItemCollections.RemoveExpiredItems()

https://forums.dfworkshop.net/viewtopic.php?t=5681
forum post of issue from 2 years back - issue still in effect.

**Comment by magicono43** [comment]
Here would be my potential suggested fix for this: 
![2024-04-11_20-01](https://github.com/Interkarma/daggerfall-unity/assets/39843717/1b4822b5-5156-4c5d-b61f-71bce35b9e8d)

That being, adding `player.UpdateEquippedArmorValues(item, false);` just under `player.ItemEquipTable.UnequipItem(slotToCheck);`

**Comment by ajrb** [comment]
Yes this is the correct way to fix this issue magicono, so submit a PR? Or I can do it if you'd rather not.

**Comment by magicono43** [comment]
> Yes this is the correct way to fix this issue magicono, so submit a PR? Or I can do it if you'd rather not.

I would but my local branch has been all screwed up for a good few months, and I haven't gotten to fixing it, lol. If you want to make that PR that would be nice.
